seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -25,6 +26,12 @@
 USER_EMAIL = 'manage_command'
 MAX_LOOKUP_VARIANTS = 5000
 
+PDO_COPY_FIELDS = [
```

Comment:
I understand what a PDO represents (a collection of samples for a project that need to be loaded into seqr) but does it stand for something?

Reply:
No idea, you would have to ask the PMs!